src/snowflake/snowpark/dataframe.py

```diff
@@ -3435,7 +3444,28 @@ def create_or_replace_dynamic_table(
             lag: specifies the target data freshness
             comment: Adds a comment for the created table. See
                 `COMMENT <https://docs.snowflake.com/en/sql-reference/sql/comment>`_.
+            mode: Specifies the behavior of create dynamic table. Allowed values are:
```

Comment:
Does a dynamic table have `truncate`? I guess no. Just to double confirm.

Reply:
It does not support it in the create or replace dynamic table api. Here we map
overwrite = CREATE OR REPLACE DYNAMIC TABLE
errorifexists = CREATE DYNAMIC TABLE
ignore = CREATE DYNAMIC TABLE IF NOT EXISTS